common/changes/@itwin/web-viewer-react/fix-proxy_2021-07-29-23-05.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/web-viewer-react",
+      "comment": "change type of WebAuthorizationOptions.oidcClient to FrontendAuthorizationClient",
+      "type": "minor"
```

Comment:
Couldnt decide between minor/patch. Feel free to disagree with what i have here and i'll change it.

Reply:
I think minor makes sense.